spec/miq_ae_engine_spec.rb

```diff
@@ -281,6 +281,15 @@ def call_automate(obj_type, obj_id, open_url_task_id = nil)
       expect(MiqAeEngine.create_automation_object("AUTOMATION", attrs, :vmdb_object => host)).to eq(uri)
     end
 
+    it "has both message from request and message from attrs" do
+      host = FactoryBot.create(:host)
+      message = "request_message"
+      attrs = {"Host::host" => host.id, :message => "attr_message"}
+      extras = "MiqServer%3A%3Amiq_server=#{@miq_server_id}"
```

Comment:
Prefer this iteration which does not add the `MiqServer` key/value in the `attrs` hash.  

We should look to refactor `@miq_server_id` along with the other instance variables at the top of this spec into `let` blocks in a separate PR.